apps/upgrade-examples-v8-v9/package.json

```diff
@@ -20,7 +20,7 @@
     "@fluentui/react": "^8.66.1",
     "@fluentui/scripts": "^1.0.0",
     "@fluentui/storybook": "^1.0.0",
-    "@fluentui/react-components": "^9.0.0-rc.6",
+    "@fluentui/react-components": "^9.0.0-rc.7",
```

Comment:
manual bump only here, since bumping this package to v9 would mean v8 bumps wouldn't happen 😔